app/components/works/show/directory_component.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if name.present? %>
+  <tr>
+    <td class="file-level" style="--depth: <%=depth%>rem" colspan="3">
+       <i class="fa fa-folder" aria-hidden="true"></i> <strong><%= name %></strong>
+    </td>
+  </tr>
+<% end %>
+<% children_files.each do |file| %>
+   <%= render Works::Show::AttachedFileComponent.new(attached_file: file.attached_file, depth:) %>
+<% end %>
+<% children_directories.each do |directory| %>
+   <%= render Works::Show::DirectoryComponent.new(directory: directory) %>
```

Comment:
It looks like line 9 is using the new keyword/hash shorthand syntax, `depth:)`, and this line is not. Consider `...(directory:)`?